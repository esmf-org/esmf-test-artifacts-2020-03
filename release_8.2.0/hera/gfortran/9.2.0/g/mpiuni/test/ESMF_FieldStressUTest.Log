build time -- 2021-10-11 06:24:04
20211011 065441.990 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211011 065441.991 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211011 065441.991 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211011 065441.991 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211011 065441.991 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211011 065441.991 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211011 065441.991 INFO             PET0 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_21-1-g86aff4e5a1
20211011 065441.991 INFO             PET0 ESMF library build date/time: "Oct 11 2021" "06:23:00"
20211011 065441.991 INFO             PET0 ESMF library build location : /scratch1/NCEPDEV/stmp2/Mark.Potts/gfortran_9.2.0_mpiuni_g_release_8.2.0
20211011 065441.991 INFO             PET0 ESMF_COMM                   : mpiuni
20211011 065441.991 INFO             PET0 ESMF_MOAB                   : enabled
20211011 065441.991 INFO             PET0 ESMF_LAPACK                 : enabled
20211011 065441.991 INFO             PET0 ESMF_NETCDF                 : enabled
20211011 065441.991 INFO             PET0 ESMF_PNETCDF                : disabled
20211011 065441.991 INFO             PET0 ESMF_PIO                    : disabled
20211011 065441.991 INFO             PET0 ESMF_YAMLCPP                : enabled
20211011 065441.991 INFO             PET0  Beginning Test, file ESMF_FieldStressUTest.F90, line 51
20211011 065441.991 INFO             PET0  NUMBER_OF_PROCESSORS           1
20211011 065441.996 INFO             PET0  PASS  Create a simple Field, repeat 10 times, ESMF_FieldStressUTest.F90, line 61
20211011 065442.030 INFO             PET0  PASS  Create a simple Field, repeat 100 times, ESMF_FieldStressUTest.F90, line 69
20211011 065442.431 INFO             PET0  PASS  Create a simple Field, repeat 1000 times, ESMF_FieldStressUTest.F90, line 77
20211011 065451.762 INFO             PET0  PASS  Create a simple Field, repeat 10000 times, ESMF_FieldStressUTest.F90, line 85
20211011 065451.762 INFO             PET0  Ending Test, file ESMF_FieldStressUTest.F90, line 88
20211011 065451.762 INFO             PET0 Finalizing ESMF
