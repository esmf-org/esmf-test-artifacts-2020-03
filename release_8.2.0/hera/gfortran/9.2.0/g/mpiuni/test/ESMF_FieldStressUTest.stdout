build time -- 2021-10-25 09:19:41
  Beginning Test, file ESMF_FieldStressUTest.F90, line 51
  NUMBER_OF_PROCESSORS           1
  PASS  Create a simple Field, repeat 10 times, ESMF_FieldStressUTest.F90, line 61
  PASS  Create a simple Field, repeat 100 times, ESMF_FieldStressUTest.F90, line 69
  PASS  Create a simple Field, repeat 1000 times, ESMF_FieldStressUTest.F90, line 77
  PASS  Create a simple Field, repeat 10000 times, ESMF_FieldStressUTest.F90, line 85
  Ending Test, file ESMF_FieldStressUTest.F90, line 88
  PET           0  Test Elapsed Time    10076.584999999999       msec.
